pages/CartPage.vue

```diff
@@ -23,12 +28,20 @@
 <!--That component should have props validation as well as use at least 4 data points
 in the template itself (heading, description, image, statistics, etc).-->
 <script>
+
+// The child page wasn't imported and added as a component.
+import childPage from '../components/ChildPage.vue'
+
 export default {
   data() {
     return {
-      mountains: []
+      mountains: [],
+      show: false
```

Comment:
This show variable was missing.  The toggle wouldn't work without it.

Reply:
I see, Aaron - thank you!